packages/auth/tsconfig.json

```diff
@@ -3,5 +3,8 @@
   "compilerOptions": {
     "rootDir": "src",
     "outDir": "dist"
-  }
+  },
+  "exclude": [
+    "dist"
```

Comment:
not sure but hmm maybe this should go for all packages in the `@standardnotes/config`?

Reply:
Good point!